presto-druid/src/test/resources/druid-tpch-ingest-nation.json

```diff
@@ -22,7 +22,7 @@
                         "dimensions": [
                             {
                                 "name": "nationkey",
-                                "type": "long"
+                                "type": "double"
```

Comment:
why?

Reply:
because the  avg behavior is different, druid seem to be doing (bigint)/(bigint) and returning the integer component so goes back to the same question as stddev functions.

Reply:
PostgreSQL does the same, so we use a cast
https://github.com/prestosql/presto/blob/bffd3cd8ba938fd4180dc3012edbfcfd48961089/presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/ImplementAvgBigint.java#L63

Reply:
Added